contracts/jobs/IJobsManager.sol

```diff
@@ -6,8 +6,4 @@ pragma solidity ^0.4.11;
  */
 contract IJobsManager {
     // External functions
-    function job(string _streamId, string _transcodingOptions, uint256 _maxPricePerSegment) external returns (bool);
-    function endJob(uint256 _jobId) external returns (bool);
-    function claimWork(uint256 _jobId, uint256 _startSegmentSequenceNumber, uint256 _endSegmentSequenceNumber, bytes32 _transcodeClaimsRoot) external returns (bool);
-    function verify(uint256 _jobId, uint256 _segmentSequenceNumber, string _dataHash, string _transcodedDataHash, bytes _broadcasterSig, bytes _proof) payable external returns (bool);
```

Comment:
None of the manager contracts currently invoke of the functions in `JobsManager` so the abstract contract `IJobsManager` doesn't need to declare any functions.